### PR TITLE
#96 Add manual testing for displaying points in 3D space

### PR DESCRIPTION
**Pull request by trangnguyen3010**
What was done and why:
- Write manual test scenarios for viewing static points in 3D space

Related Feature Development: https://github.com/UniversityOfSaskatchewanCMPT371/term-project-2024-team-2/issues/79

### CODE REVIEW

**Review by LongQuanNguyen — APPROVED** [review]
This looks good. You may make use of given, this would look something like:

Given the app is loaded on the Quest 2
When the data point is rendered in 3D space
And user walk around the data points
Then data point should be visible form all angle.

File: Test/GherkinFeatureFiles/display-points.feature [hunk 1]
```diff
@@ -0,0 +1,46 @@
+Feature: Data Point Component in WebXR Application
+  Background:
+    Given That the user is on the Meta Quest 2
```
Comment (MitchWag01) [inline Test/GherkinFeatureFiles/display-points.feature:3]: Can we move the WebXR application is loaded into the background?
something like
  Background:
    Given that the user is on a Meta Quest 2
    And they are inside of the quest browser
    And they have loaded the web app

File: Test/GherkinFeatureFiles/display-points.feature [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  Scenario: View Data Points in 3D Space
+    When the WebXR application is loaded
+    Then the data points should be rendered in 3D space
+
+  Scenario: View Data Points in 3D Space while walking
+    When the WebXR application is loaded
```
Comment (MitchWag01) [inline Test/GherkinFeatureFiles/display-points.feature:10]: if you want to keep all of these as When the WebXR application is loaded (which you can, then ignore my prior comment) then all you have to do is change this to say Given the WebXR application is loaded

**Review by MitchWag01 — COMMENTED** [review]
It looks good, you have really good test coverage. just have it use the given -> when -> then, syntax

**Review by Subzeero — APPROVED** [review]
I think these are really nice and thorough. Just the same comment Mitch made, then I'd say it's good 👍

**Pull request merged**